### PR TITLE
Trivia

### PR DESCRIPTION
**Pull request by YoussefExplicit**
In the trivia command, if multiple people request the command, it wasn't clear on who requested what, simply added a .setFooter to the embed to display the author's tag and avatar.

### CODE REVIEW

File: commands/fun/trivia.js [hunk 1]
```diff
@@ -36,7 +36,8 @@ class Trivia extends Social {
       .addField("A", randomChoices[0])
       .addField("B", randomChoices[1])
       .addField("C", randomChoices[2])
-      .addField("D", randomChoices[3]);
+      .addField("D", randomChoices[3])
+      .setFooter(`Requested by: ${message.author.tag}`, message.author.displayAvatarURL());
```
Comment (YorkAARGH) [inline commands/fun/trivia.js]: We're just nit-picking, could you put the format to `png` and a size of 64 pls

**Comment by YoussefExplicit** [comment]
fixed ;)

File: commands/fun/trivia.js [hunk 2]
```diff
@@ -36,7 +36,8 @@ class Trivia extends Social {
       .addField("A", randomChoices[0])
       .addField("B", randomChoices[1])
       .addField("C", randomChoices[2])
-      .addField("D", randomChoices[3]);
+      .addField("D", randomChoices[3])
+      .setFooter(`Requested by: ${message.author.tag}`, message.author.displayAvatarURL({format: 'png', size: 64}));
```
Comment (YorkAARGH) [inline commands/fun/trivia.js]: Misaki uses double quotes "

**Review by YoussefExplicit — COMMENTED** [review]
added double quotes

**Pull request merged**